output/transform/postgres_relations.go

```diff
@@ -63,15 +63,38 @@ func transformPostgresRelations(s snapshot.FullSnapshot, newState state.Persiste
 			Options:                relation.Options,
 		}
 
+		schemaStats, schemaStatsExist := newState.SchemaStats[relation.DatabaseOid]
+
 		if relation.ViewDefinition != "" {
 			info.ViewDefinition = &snapshot.NullString{Valid: true, Value: relation.ViewDefinition}
 		}
 		for _, column := range relation.Columns {
+			var stats []*snapshot.RelationInformation_ColumnStatistic
+			if schemaStatsExist {
+				key := relation.SchemaName + "\t" + relation.RelationName + "\t" + column.Name
```

Comment:
Per the note in the state file, I think it would be better to use a strongly typed key (i.e. have a struct as the key instead of a string), similar to how we do it for statements.